hi-ml-histopathology/src/histopathology/utils/naming.py

```diff
@@ -32,6 +32,10 @@ class TileKey(str, Enum):
     SPLIT = 'split'
     TILE_X = 'tile_x'
     TILE_Y = 'tile_y'
```

Comment:
Maybe add a TODO to remove TILE_X. TILE_Y once all the datasets have been recreated according to this new philosophy